### PR TITLE
created order-list page and added a order here button

### PR DESCRIPTION
**Pull request by henkegithub**
Created the order-list page and added an "Order Here!"-Button which directs to the order-list page

### CODE REVIEW

**Review by ekat-br — APPROVED** [review]
lgtm

File: pages/components/order-button/index.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import Link from "next/link";
+
+export default function OrderButton() {
+  return (
+    <div>
+      <button>
+        <Link href="/order-list">Order Here!</Link>
+      </button>
+    </div>
```
Comment (CJAkkuec) [inline pages/components/order-button/index.js:9]: You don't need the div or the button element here. In fact, a link should always just be a link. You can style it to have the "look and feel" of a button, but semantically anything that routes people somewhere is a link, and anything that does nothing but add interaction without directing people elsewhere is just a button.

File: pages/order-list/index.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export default function OrderList() {
+  console.log("Order List");
```
Comment (CJAkkuec) [inline pages/order-list/index.js:2]: Whenever you open a PR, make sure that you don't have commented out code or `console.log`s remaining in your code. It makes reviews a little less "cluttered" with things people don't need to look at.

**Review by CJAkkuec — CHANGES_REQUESTED** [review]

**Review by Roland-Hufnagel — APPROVED** [review]
LGTM

**Pull request merged**